src/Lightning.ts

```diff
@@ -290,6 +291,23 @@ export const LightningMixin = (superclass) =>
         params,
       })
 
+      if (
+        yamlConfig.twoFactor?.enabled &&
```

Comment:
what is the goal of this settings?

Reply:
Deploy PR with this disabled, make sure nothing breaks, then enable this

Reply:
> Deploy PR with this disabled, make sure nothing breaks, then enable this.

By default this is not enabled at the user level? So deploying this will be a no-op for a user perspective until he decides to activate it (once we have the integration in the mobile app obviously). 

> Could also be used as a switch to disable any 2FA related stuff across the system (Maybe someone doesn't want to deploy galoy with 2FA?)

I think this is unlikely once we have a good 2FA system, but this is a good point. I think eventually there should be settings to configure the auth part of galoy.

Reply:
> > Deploy PR with this disabled, make sure nothing breaks, then enable this.
> 
> By default this is not enabled at the user level? So deploying this will be a no-op for a user perspective until he decides to activate it (once we have the integration in the mobile app obviously).

Yes, that's correct. My first point is moot.

> > Could also be used as a switch to disable any 2FA related stuff across the system (Maybe someone doesn't want to deploy galoy with 2FA?)
> 
> I think this is unlikely once we have a good 2FA system, but this is a good point. I think eventually there should be settings to configure the auth part of galoy.

Yes, which is why I didn't remove this. Do you want this to be removed for now and added back later?

If we do want to do this in the future then I think we should leave this here and if we end up adopting a different kind of switching approach (something other than the yaml based approach we're using currently for switches) then we can refactor this.

Reply:
I think a more elegant way to do it could be to have `remainingTwoFactorLimit` returning infinite (or MaxInt) when it's not enabled. this would avoid this additional if here. (the pay function is already big we should try to not increase it's size too much)

Reply:
@krtk6160 I think that should be considered/addressed

Reply:
@krtk6160 this is my last comment on this PR

Reply:
I've completely removed the system level limit